src/traces/waterfall/attributes.js

```diff
@@ -96,7 +96,9 @@ module.exports = {
     connector: {
         line: {
             color: extendFlat({}, lineAttrs.color, {dflt: Color.defaultLine}),
-            width: lineAttrs.width,
+            width: extendFlat({}, lineAttrs.width, {
```

Comment:
does a similar fix need to be applied anywhere else in this trace? I haven't rigorously tested each attr, I just happened to test this one a lot due to our conversations around the line extensions etc :)